tests/snapshot_tests/test_snapshots.py

```diff
@@ -333,6 +333,10 @@ def test_select(snap_compare):
     assert snap_compare(WIDGET_EXAMPLES_DIR / "select_widget.py")
 
 
+def test_select_from_values(snap_compare):
```

Comment:
I don't think we need quite so many snapshot tests here. These tests are for higher level things, so we shouldn't have multiple snapshots for a single method on a single widget.

Reply:
Do you suggest we keep just `test_select_from_values_expanded` and delete the other 4?

Reply:
I think so, yeah.

Reply:
i've updated the snapshot tests and snapshots accordingly